app/assets/javascripts/tree_display.jsx

```diff
@@ -464,6 +464,22 @@ jQuery(document).ready(function() {
         expanded: false
       }
     },
+      componentDidMount: function() {
+          rubBuffer = ["Review", "Metareview", "Author Feedback", "Teammate Review", "Assignment Survey", "Global Survey", "Course Survey"];
+
+          selMenuItem = document.getElementById("tree_display").getAttribute("data-menu-item");
+          selMenuItemInd = rubBuffer.indexOf(selMenuItem);
+
+          if(rubBuffer[selMenuItemInd] === this.props.name) {
+              if (selMenuItemInd !== -1) {
```

Comment:
Please put this check before the check on line number 473. If `selMenuItemInd ` is in fact `-1` the condition would be useless because your code tried accessing using -`1` as index anyway.